_pytest/config.py

```diff
@@ -636,7 +636,7 @@ def __init__(self, *names, **attrs):
                 'pytest now uses argparse. "%default" should be'
                 ' changed to "%(default)s" ',
                 DeprecationWarning,
-                stacklevel=3)
```

Comment:
this change looks incorrect, the argument type is used indirectly and the higher level intended to trigger that warning for the method call side